packages/app-store/ee/routing-forms/components/FormActions.tsx

```diff
@@ -109,14 +109,14 @@ function NewFormDialog({ appUrl }: { appUrl: string }) {
             });
           }}>
           <div className="mt-3 space-y-4">
-            <TextField label={t("title")} required placeholder="A Routing Form" {...register("name")} />
```

Comment:
Strings are still not added to `apps/web/public/static/locales/en/common.json`

Reply:
okay, so whatever the strings that are needed to be translated, do we need to add them to the common.json file?

Reply:
yes, new strings need to be added to /en/common.json

the rest will get translated via our translation platform

Reply:
done